sqlparser/sql.y

```diff
@@ -861,6 +869,10 @@ char_type:
   {
     $$ = ColumnType{Type: string($1), Length: $2}
   }
+| BYTEA length_opt
+  {
+    $$ = ColumnType{Type: string($1), Length: $2}
```

Comment:
does postgresql accepts length for bytea? should it have `length_opt`? 
https://www.postgresql.org/docs/current/datatype-binary.html